src/app/authoring-tool/add-component/choose-new-component/choose-new-component.component.html

```diff
@@ -16,18 +16,12 @@ <h5 i18n>
     </div>
   </div>
   <div fxLayout="row wrap" fxLayoutAlign="start center">
-    <button
-      mat-stroked-button
-      *ngFor="let componentType of componentTypes"
-      class="mat-card component-type"
-      (click)="selectComponent(componentType.type)"
-    >
-      <mat-card-content>
-        <mat-card-title>
-          {{ componentType.name }}
-        </mat-card-title>
-      </mat-card-content>
-    </button>
+    <ng-container *ngFor="let componentType of componentTypes">
+      <component-selector
+        [componentType]="componentType.type"
+        (componentSelectedEvent)="selectComponent($event.type)"
```

Comment:
We already know the component type at this point, so no need to get it from $event?
```suggestion
        (componentSelectedEvent)="selectComponent(componentType.type)"
```